src/status_im/ui/screens/wallet/accounts/views.cljs

```diff
@@ -149,13 +149,17 @@
        [icons/icon :main-icons/more {:accessibility-label :accounts-more-options}]]]]))
 
 (views/defview accounts []
-  (views/letsubs [{:keys [accounts]} [:multiaccount]]
-    [react/scroll-view {:horizontal true}
-     [react/view {:flex-direction :row :padding-top 11 :padding-bottom 12}
-      (for [account accounts]
-        ^{:key account}
-        [account-card account])
-      [add-card]]]))
+  (views/letsubs [{:keys [accounts address]} [:multiaccount]
+                  multiaccounts [:multiaccounts/multiaccounts]]
```

Comment:
once login you should never rely on this subscription, it should even be dissoc at login in theory, why isn't keycard-key-uid stored in multiaccount as well?

Reply:
@yenda right, i'd rather pull `keycard-key-uid` from `multiaccount`, but it's not there. Do you think we should add that once data arrives from status-go side in `get-config-callback`?

Reply:
There is two options:
- it should be passed through `:multiaccount/login` which is dissoced in `multiaccount-login-success`
- or as you said, which is done for other stuff that is duplicated from multiaccounts/multiaccounts in the multiaccount settings but it's not correct, I have thought about fixing that for a while but didn't had time

Reply:
@yenda I just checked and there is `keycard-key-uid` value in `multiaccount` once you login. So i'm just getting that value from there, not from `multiaccounts/multiaccounts`